lib/events.js

```diff
@@ -118,6 +118,21 @@ function checkListener(listener) {
   }
 }
 
+function checkType(type) {
+  // Besides string/symbol also numeric types and null are seen in the wild
+  switch (typeof type) {
+    case 'string':
+    case 'symbol':
+    case 'number':
+    case 'bigint':
+      return;
+  }
+  if (type === null) {
```

Comment:
Supporting `null` does not seem a good idea to me. It is very likely a bug, if this happens in any application.

Reply:
I thought the same but there is at least test-event-emitter-max-listeners-warning-for-null.js which changed my mind.

Reply:
The reason was a comment that all types are allowed and a test should guarantee it should not fail somewhere else in case such value is passed through: https://github.com/nodejs/node/pull/9021#discussion_r82740119

I think it's safe to exclude this. We should of course also run CITGM and we could also use https://github.com/nodejs/Gzemnid to check in more depth.

Reply:
done